src/gui_graphics.cpp

```diff
@@ -219,13 +217,14 @@ static uint8 *GetSingleLine(uint8 *text, int max_width, int *width)
  */
 void GetMultilineTextSize(StringID strid, int max_width, int *width, int *height)
 {
-	uint8 buffer[1024]; // Arbitrary max size.
-	DrawText(strid, buffer, lengthof(buffer));
+	/* \todo This design is ugly. Fix the utility function GetSingleLine to work on std::string with a srart index instead of mutable char* . */
```

Comment:
```suggestion
	/* \todo This design is ugly. Fix the utility function GetSingleLine to work on std::string with a start index instead of mutable char* . */
```